editions/test/tiddlers/tests/test-checkbox-widget.js

```diff
@@ -119,6 +119,25 @@ Tests the checkbox widget thoroughly.
             return newData;
         });
 
+        const listModeTestsForDateFields = [
```

Comment:
We do not use `const` yet so go with `var`. ... We still use ES5 atm.

Reply:
I have `const` throughout that file. I'll switch it all to `var` in a new commit, so this PR will end up touching a lot more lines than just to fix the bug. However, I'll keep that const->var switch in its own commit, so it's easy to back off if the decision ends up to leave it as-is.

Reply:
oh... It's the tests. I didn't see that. .. May be it's OK to use if for tests, since that does not make a difference for the users.

Reply:
Done in commit eb3e31dc5.

Reply:
Note that there are several plugins using `const`: the highlight.js syntax highlighting plugin has `const` throughout the library file that it pulls in, and the Twitter Archivist plugin also has `const` sprinkled throughout the code. So if we want to really be strict about ES5 targeting, we'd need to change those as well. https://caniuse.com/?search=const suggests that it's only a problem for people using IE 10 or below, so hopefully nobody at all. Still, I changed it as requested.